client/src/components/Container.jsx

```diff
@@ -52,44 +51,35 @@ class Container extends React.Component {
   }
 
   getData() {
-    var endpoint = window.location.pathname + 'store_id';
+    const endpoint = 'http://13.52.66.18' + window.location.pathname + 'store_id';
```

Comment:
while the usage of `const` is OK here (it functionally works), this should be `let` since it's a scope bound variable to `getData()`.  `const` is used for variables that are never reassigned.  In the functional context here, the variable could be reassigned on a second call of the function (of course, in this case, the only time the function is loaded is on page load)